graphql/src/execution/execution.rs

```diff
@@ -178,46 +199,17 @@ where
             query: self.query.as_introspection_query(),
             deadline: self.deadline,
             max_first: std::u32::MAX,
-            cached: AtomicBool::new(false),
+            cached: AtomicBool::new(true),
```

Comment:
Don't get the changes on `cached` in this PR.

Reply:
The `QueryCache` isn't aware of the `ctx` and therefore doesn't set `ctx.cached`. Rather than make every cache layer aware of this variable just move the setting of cached to false in one spot where the query is in fact executed.

Reply:
Ah makes sense. The comment on the declaration of this field should be updated.